### PR TITLE
Dev 2205

### PR DESCRIPTION
**Pull request by Slugger70**

### CODE REVIEW

**Review by jlqfab — COMMENTED** [review]
I'm not sure what the uwsgi configuration changes (additions) do that the existing settings or defaults did, so I won't comment on these. The rest appears to mostly be code formatting cleanup, and looks fine to me.

**Review by jlqfab — COMMENTED** [review]
based on what I know the commit is intended to do, this looks ok to me. Someone (Nuwan) with more in depth knowledge of Galaxy should also review, to do the approval.

File: templates/nginx/galaxy-dev.j2 [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        upload_pass_args on;
+        upload_pass /_upload_job_files_done;
+    }
+
+    location /_upload_job_files_done {
+        internal;
+        rewrite "" /api/jobs/$arg_job_id/files;
+    }
+
+{% endif %}
```
Comment (nuwang) [inline templates/nginx/galaxy-dev.j2:156]: Can this section be removed now?
```suggestion
```

File: host_vars/dev.usegalaxy.org.au.yml [hunk 2]
```diff
@@ -40,13 +40,17 @@ galaxy_dynamic_job_rules:
...
 galaxy_custom_indices_dir: "{{ galaxy_root }}/custom-indices"
 galaxy_tmp_dir: "{{ galaxy_root }}/tmp"
 galaxy_tus_upload_store: "{{ galaxy_tmp_dir }}/tus"
 
 galaxy_repo: https://github.com/galaxyproject/galaxy.git
-galaxy_commit_id: release_22.01
+galaxy_commit_id: release_22.05
 
 galaxy_file_path: "{{ galaxy_root }}/data-3"
 nginx_upload_store_base_dir: "{{ galaxy_file_path }}/upload_store"
```
Comment (nuwang) [inline host_vars/dev.usegalaxy.org.au.yml:56]: Remove in favour of tusd?

File: templates/nginx/galaxy-dev.j2 [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    uwsgi_busy_buffers_size 24k;
+
+   # The most important location block, by default all requests are sent to Gunicorn
+    location / {
+        proxy_pass         http://unix://{{ galaxy_mutable_config_dir }}/gunicorn.sock;
+        proxy_redirect     off;
+        proxy_set_header   Host              $host;
+        proxy_set_header   X-Real-IP         $remote_addr;
+        proxy_set_header   X-Forwarded-For   $proxy_add_x_forwarded_for;
+        proxy_set_header   X-Forwarded-Proto $scheme;
```
Comment (nuwang) [inline templates/nginx/galaxy-dev.j2:22]:
```suggestion
        proxy_set_header   X-Forwarded-Proto $scheme;
        proxy_set_header   Upgrade $http_upgrade;
```

File: templates/nginx/galaxy-dev.j2 [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    # location ~ ^/api/dataset_collections/([^/]+)/download/?$ {
+    #     proxy_buffering off;
+    #     uwsgi_pass 127.0.0.1:8080;
+    #     uwsgi_param UWSGI_SCHEME $scheme;
+    #     include uwsgi_params;
+    # }
+
+    location /_x_accel_redirect/ {
+        internal;
+        alias /;
```
Comment (nuwang) [inline templates/nginx/galaxy-dev.j2:57]:
```suggestion
        alias /;
        add_header X-Content-Type-Options nosniff;
```

**Review by nuwang — APPROVED** [review]
LGTM! Some minor comments.

File: templates/nginx/galaxy-dev.j2 (hunk 1 above)
Comment (Slugger70) [reply templates/nginx/galaxy-dev.j2:156]: I want to do one thing at a time. And I thni kwe are stulll using nginx uploads for job returns from pulsar atm.

File: host_vars/dev.usegalaxy.org.au.yml (hunk 2 above)
Comment (Slugger70) [reply host_vars/dev.usegalaxy.org.au.yml:56]: see above comment

**Comment by cat-bro** [comment]
Do staging/aarnet/pawsey now need to have `galaxy_manage_systemd: yes` in their host vars?

**Comment by Slugger70** [comment]
yes they do

**Comment by Slugger70** [comment]
But having it in galaxyservers.yml is fine as it's over-rideable at the host var level

**Pull request merged**